tools/phenomenal/w4m/update-tools

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
You should be able, I could when Luca was reviewing my last push and asked for some corrections. I think that you tried to push the version that you already locally merged to your local develop... you would need a git reflog to go backwards before that merge... I'll give it a try, I got into the same trap on my PR.